modules/ROOT/pages/dataweave-maven-plugin.adoc

```diff
@@ -0,0 +1,329 @@
+= Packaging and Deploying DataWeave Libraries
+
+Use the DataWeave Maven plugin to integrate the packaging and deployment of your DataWeave libraries with your Maven lifecycle.
+
+The DataWeave Maven plugin has six main goals you can run:
+
+* `compile` +
+Compiles the source code for your DataWeave library.
+
+* `test` +
+Tests the compiled source code using the https://github.com/mulesoft/data-weave-testing-framework[DataWeave Testing Framework].
+
+* `data-weave:generate-docs` +
+Automatically generates the documentation for your DataWeave library.
+
+* `package` +
+Packages the compiled code in a `JAR` file.
+
+* `data-weave:deploy-docs` +
+Uploads the auto-generated documentation for your DataWeave library to Anypoint Exchange.
+
+* `deploy` +
+Automatically uploads your DataWeave library to the deployment target and uploads the auto-generated documentation to Exchange.
+
+== Add the DataWeave Maven Plugin to a Project
+
+To enable the DataWeave Maven plugin in your project:
```

Comment:
```suggestion
To enable the Maven plugin for DataWeave in your project:
```
I know that our legacy doc refers to the Mule Maven plugin, but I think we should change that, going forward, to (Apache) Maven plugin for Mule as well.

Reply:
I will stop changing the name for the rest of this PR review, in case you object to changing it.